brigade/templates/base.html

```diff
@@ -16,39 +14,58 @@
     <!-- FontAwesome Icons -->
     <link href="https://use.fontawesome.com/releases/v5.0.4/css/all.css" rel="stylesheet">
 
+    <!-- Font Vendor Stylesheets -->
+    <script src="https://use.typekit.net/lkd6vsz.js"></script>
```

Comment:
I'll try to find out how to access our Typekit settings -- I think it was setup specifically for www.codeforamerica.org and sandbox.codeforamerica.org, so might not work properly on brigade.codeforamerica.org without adding the subdomain domain to the Typekit settings.